cluster-autoscaler/core/scaledown/budgets/budgets_test.go

```diff
@@ -298,6 +362,62 @@ func TestCropNodesToBudgets(t *testing.T) {
 			wantEmpty:                generateNodeGroupViewList(testNg, 0, 4),
 			wantDrain:                generateNodeGroupViewList(testNg, 0, 2),
 		},
+		"empty&drain atomic nodes with deletions in progress, overall budget exceeded, both empty&drain nodes fit": {
```

Comment:
>both empty&drain nodes fit

nit: it looks like only empty nodes fit in this test case? Unless I'm misunderstanding something :)

Reply:
You are right, fixed it :)